src/main/java/edu/ucsb/cs156/rec/controllers/RecommendationRequestController.java

```diff
@@ -146,7 +146,7 @@ public RecommendationRequest updateRecommendationRequestAsAdmin(
     @GetMapping("/requester/all")
     public Iterable<RecommendationRequest> allRequesterRecommendationRequests(
     ) {
-        // toyed with having this only be ROLE_STUDENT but I think even professors should be able to submit requests so they can see which ones they have submitted too
+        // Professors should be able to submit requests so they can see which ones they have submitted to
```

Comment:
I think in the end it's probably better to just remove this comment; I'm not sure it's helping anything at this point.  You can just click "commit suggestion" if you agree.

```suggestion
```